### DISCUSSION TITLE
How to turn off slected text turning reverse?

### DISCUSSION THREAD

**Discussion by Alex1231273** [Godiesc/firefox-gx#187] [Q&A] [answered]
![Screenshot 2024-07-26 185549](https://github.com/user-attachments/assets/23bfa8a7-c545-4328-8675-d6684660b0c0)

It feels too bright (or dark? Idk how to explain). How to turn it off so that selected text remains white?

**Comment by Godiesc** [comment, marked as answer]
hi,
you can edit the lines 711 and 714 from the file [ogx_UC-newtabpage.css](https://github.com/Godiesc/firefox-gx/blob/main/chrome/components/ogx_UC-newtabpage.css#L711-L714)

**Reply by Alex1231273** [reply]
It works, ty.
